src/pages/DistributionsPage/DistributionsPage.tsx

```diff
@@ -100,21 +101,35 @@ export function DistributionsPage() {
       d.distribution_type === DISTRIBUTION_TYPE.COMBINED
   );
 
-  const isCombinedDistribution =
-    fixedDist &&
-    circleDist &&
-    circleDist.distribution_type === DISTRIBUTION_TYPE.COMBINED &&
-    fixedDist.distribution_type === DISTRIBUTION_TYPE.COMBINED;
-
+  const getUserClaimedFixedPaymentAmt = (
+    dist: typeof epoch.distributions[0],
+    address?: string
+  ) => {
+    if (!address) return 0;
+    return dist.distribution_json?.fixedGifts &&
+      address in dist.distribution_json.fixedGifts
+      ? Number(
+          formatUnits(
+            dist.distribution_json.fixedGifts[address],
+            dist.vault.decimals
+          )
+        )
+      : 0;
+  };
   const unwrappedAmount = (
     id?: number,
     dist?: typeof epoch.distributions[0]
   ) => {
-    if (!id || !dist) return 0;
-    return (
-      (dist.claims.find(c => c.profile?.id === id)?.new_amount || 0) *
-      dist.pricePerShare.toUnsafeFloat()
-    );
+    if (!id || !dist) return { claimed: 0, fixedPayment: 0, circleClaimed: 0 };
+    const claim = dist.claims.find(c => c.profile?.id === id);
+    const pricePerShare = dist.pricePerShare.toUnsafeFloat();
```

Comment:
This is the case across our app I supposed, but why not convert this straight from a BigNumber to a String? It doesn't seem like it'd be a much bigger lift to make this more robust by avoiding the lossy conversion to a JS `Number` type.